`make html` not working in nix development environment

**Description:**

When working in nix dev shell (activated by `nix develop`) I cannot run the `make html` command to generate the documentation as described in the README (https://github.com/arbeitszeit/arbeitszeitapp?tab=readme-ov-file#documentation)

**Details:**

Output:

```bash
❯ make html
Traceback (most recent call last):
  File "/nix/store/1l5mygrbwwcr9d15wh0bd593pzdbng68-python3.11-sphinx-7.2.6/bin/.sphinx-build-wrapped", line 9, in <module>
    sys.exit(main())
             ^^^^^^
  File "/nix/store/1l5mygrbwwcr9d15wh0bd593pzdbng68-python3.11-sphinx-7.2.6/lib/python3.11/site-packages/sphinx/cmd/build.py", line 326, in main
    locale.setlocale(locale.LC_ALL, '')
  File "/nix/store/asiphbpiy2gmidfm3xbwcikayhs66289-python3-3.11.7/lib/python3.11/locale.py", line 627, in setlocale
    return _setlocale(category, locale)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
locale.Error: unsupported locale setting
make: *** [Makefile:20: html] Error 1
```

Outout of `locale` command (also in nix develop shell):

```bash
❯ locale
locale: Cannot set LC_CTYPE to default locale: No such file or directory
locale: Cannot set LC_MESSAGES to default locale: No such file or directory
locale: Cannot set LC_ALL to default locale: No such file or directory
LANG=de_DE.UTF-8
LC_CTYPE="de_DE.UTF-8"
LC_NUMERIC="de_DE.UTF-8"
LC_TIME="de_DE.UTF-8"
LC_COLLATE="de_DE.UTF-8"
LC_MONETARY="de_DE.UTF-8"
LC_MESSAGES="de_DE.UTF-8"
LC_PAPER="de_DE.UTF-8"
LC_NAME="de_DE.UTF-8"
LC_ADDRESS="de_DE.UTF-8"
LC_TELEPHONE="de_DE.UTF-8"
LC_MEASUREMENT="de_DE.UTF-8"
LC_IDENTIFICATION="de_DE.UTF-8"
LC_ALL=
```

When I am NOT in the nix environment, I can run the command and generate the docs. 

I am not sure if this is a problem of my local setup. Can you reproduce it?

--- a/nix/devShell.nix
+++ b/nix/devShell.nix
@@ -1,4 +1,4 @@
-{ mkShell, python3, nixfmt, sqlite }:
+{ mkShell, python3, nixfmt, sqlite, glibcLocales }:
 mkShell {
   packages = (with python3.pkgs; [
     black
@@ -16,4 +16,5 @@ mkShell {
   ]) ++ [ nixfmt sqlite ]
     ++ python3.pkgs.arbeitszeitapp.optional-dependencies.profiling;
   inputsFrom = [ python3.pkgs.arbeitszeitapp ];
+  LOCALE_ARCHIVE = "${glibcLocales}/lib/locale/locale-archive";
 }

